Tests/Common/Data/RenkoConsolidatorTests.cs

```diff
@@ -737,5 +737,24 @@ public void WickedTwoRisingThenThreeFallingGapRenkos()
             Assert.AreEqual(openRenko.Low, 7.9);
             Assert.AreEqual(openRenko.Close, 7.9);
         }
+
+        [Test]
+        public void WickedArgumentExceptionRenkoTypeClassicTest()
+        {
+            Assert.Throws<ArgumentException>(() => new RenkoConsolidator(2.5m, RenkoType.Classic));
```

Comment:
This API choice concerns me; I think it would be better to split this into a class per Renko or rethink the API for this class so the user can't accidentally cause an explosion by putting a perfectly valid looking arg into the constructor.

Reply:
This whole class is a candidate for deprecation and replacement with two separate classes; maybe making the old class inherit from its replacement to maintain backward compatibility.

Reply:
Agree 👍, if `RenkoType` can only have 1 value then it doesn't seem like an argument at all